### ISSUE TITLE
Screen flashes when pasting text into XCL Exec in Medley Online

### ISSUE THREAD

**Issue by pamoroso** [Interlisp/medley#1595]
**Describe the bug**
In Medley Online, when I paste into a XCL Exec with `Alt+V` some text from the noVNC clipboard, the screen flashes (i.e. Medley briefly inverts black pixels with white and vice versa) and no text is pasted. Pasting works fine with Interlisp Execs.

**To Reproduce**
1. sign into your Interlisp Online account
2. under `Initial Exec`, select `Interlisp`
3. click `Run Medley`
4. open a XCL Exec
5. from the noVNC menu, open the Clipboard
6. copy some text from the host system and paste it into the noVNC Clipboard
7. click the XCL Exec to set the focus
8. press `Alt+V`

**Expected behavior**
The text in the noVNC Clipboard appears in the XCL Exec.

**Screenshots**
N/A

**Context (please complete the following information):**
 - OS: chromeOS Stable
 - OS Version: 122.0.6261.118
 - Host arch: x86_64 (ASUS Chromebox 3)
 - Maiko version: N/A
 - IL:MAKESYSDATE: 19-Mar-2024 02:06:49

**Additional context**
This is a fresh build of Medley Online and the issue didn't happen with previous builds.

**Comment by pamoroso** [comment]
I don't know whether it's related but now from a XCL Exec CLOS symbols have to be qualified when the `XCL-USER` package is current. For example:

```lisp
2/9> (defclass myclass ())
Undefined car of form
DEFCLASS

2/10> (clos:defclass myclass ())
MYCLASS
```

I'm reasonably sure unqualified CLOS symbols could be used up to yesterday, i.e. prior to the 19-Mar-2024 02:06:49 build.

**Comment by rmkaplan** [comment]
This issue came up in the technical meeting on Monday.

Some people reported it working in an Interlisp exec but not an XCL, some reported it working in XCL but not Interlisp.

CLIPBOARD modifies the special input tables for Sedit and Tedit, but otherwise pasting is supposed to be accomplished by setting meta-v as an interrupt character that inserts into the current TTY process.

So presumably the interrupts are not propagating consistently.

**Comment by pamoroso** [comment]
It may be some change introduced since the previous online build, i.e. within the past month or so, as I've never experienced such an issue since using Medley Online.

**Comment by nbriggs** [comment]
Is "The Wednesday meetings are for check-in and discussion of external communications (Website, documentation, talks, videos, social media etc.)" no longer accurate?

**Comment by masinter** [comment]
Still scoping out what the problem is. I'm leaning toward redoing interrupt character assignment so that users have a simple interface for changing their key bindings -> control character management.
And getting rid of the MOVD? calls and instead use a global ALIST that TEdit and S-Edit can add to using he ALISTS file package command.

**Comment by masinter** [comment]
I tried adding (INSTALL-CLIPBOARD) at the end of (LOADUP-FULL) and it still had clispboard interrupts disabled. That should be a clue, but in the meanwhile, this patches it for everyone.

**Comment by rmkaplan** [comment]
Maybe the clipboard isntaller needs to go on AFTERSYSOUTFORMS and AFTERMAKESYSFORMS ?

**Comment by masinter** [comment]
The two problems don't seem to be related. The meta-V interrupt missing in some shells is due to the way the interrupt list is initialized, and is patched by #1626.  The cl package problem is an interaction of bringing the LISP package up, and is fixed by #1695.
